src/components/categories.jsx

```diff
@@ -0,0 +1,4 @@
+const Categories = () => (
+  <div />
+);
+export default Categories;
```

Comment:
- [ ] `required`
- [ ] please make sure the category has a `button` which renders  `Check status`

Reply:
Hello Sir..
But rember that they told us categories have to be emptyin first projct of bookstore
![Screenshot (51)](https://user-images.githubusercontent.com/79504482/229296601-dddb941a-c522-48a0-990d-b7e6f15f1669.png)